### PR TITLE
Merger

### PR DESCRIPTION
**Pull request by harry-bridge**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/newtheatre/nt-tickets/pull/1" target="_blank">ReviewNinja

File: NT_Tickets/REQUIREMENTS.txt [hunk 1]
```diff
@@ -1,9 +1,11 @@
-Django==1.8.6
```
Comment (harry-bridge) [inline NT_Tickets/REQUIREMENTS.txt]: updated dependencies

Comment (harry-bridge) [inline NT_Tickets/REQUIREMENTS.txt]: !fix

Comment (harry-bridge) [inline NT_Tickets/REQUIREMENTS.txt]: !fixed

**Pull request merged**